src/Language/PureScript/Parser/Lexer.hs

```diff
@@ -562,6 +562,7 @@ reservedPsNames = [ "data"
                   , "infix"
                   , "class"
                   , "instance"
+                  , "instances"
```

Comment:
Oh I can revert this change.  I updated the syntax to use interleaved `else` - which are already in this list.